.travis.yml

```diff
@@ -1,15 +1,18 @@
-language: scala
+language: java
 sudo: false
 jdk:
   - oraclejdk8
-script:
-  - sbt +test
+env:
+  matrix:
+    - SCRIPT="mvn test"
+    - SCRIPT="sbt test"
 cache:
   directories:
-    - $HOME/.ivy2/cache
```

Comment:
Thanks for catching this @TimMoore - I missed it when reapplying the sbt build settings to `.travis.yml`.